packages/evm/src/precompiles/bank.ts

```diff
@@ -0,0 +1,238 @@
+	{
+		inputs: [
+			{ internalType: 'address', name: 'acc', type: 'address' },
+			{ internalType: 'string', name: 'denom', type: 'string' }
+		],
+		name: 'balance',
+		outputs: [{ internalType: 'uint256', name: 'amount', type: 'uint256' }],
+		stateMutability: 'view',
+		type: 'function'
+	},
+	{
+		inputs: [{ internalType: 'string', name: 'denom', type: 'string' }],
+		name: 'decimals',
+		outputs: [{ internalType: 'uint8', name: 'response', type: 'uint8' }],
+		stateMutability: 'view',
+		type: 'function'
+	},
+	{
+		inputs: [{ internalType: 'string', name: 'denom', type: 'string' }],
+		name: 'name',
+		outputs: [{ internalType: 'string', name: 'response', type: 'string' }],
+		stateMutability: 'view',
+		type: 'function'
+	},
+	{
+		inputs: [
+			{ internalType: 'address', name: 'fromAddress', type: 'address' },
+			{ internalType: 'address', name: 'toAddress', type: 'address' },
+			{ internalType: 'string', name: 'denom', type: 'string' },
+			{ internalType: 'uint256', name: 'amount', type: 'uint256' }
+		],
+		name: 'send',
+		outputs: [{ internalType: 'bool', name: 'success', type: 'bool' }],
+		stateMutability: 'nonpayable',
+		type: 'function'
+	},
+	{
+		inputs: [{ internalType: 'string', name: 'denom', type: 'string' }],
+		name: 'supply',
+		outputs: [{ internalType: 'uint256', name: 'response', type: 'uint256' }],
+		stateMutability: 'view',
+		type: 'function'
+	},
+	{
+		inputs: [{ internalType: 'string', name: 'denom', type: 'string' }],
+		name: 'symbol',
+		outputs: [{ internalType: 'string', name: 'response', type: 'string' }],
+		stateMutability: 'view',
+		type: 'function'
+	},
+	{
+		inputs: [{ internalType: 'string', name: 'toNativeAddress', type: 'string' }],
+		name: 'sendNative',
+		outputs: [{ internalType: 'bool', name: 'success', type: 'bool' }],
+		stateMutability: 'payable',
+		type: 'function'
+	}
+] as const;
+
+/**
+ * Creates and returns an ethers v6 contract instance with the provided signer, for use in interoperability between the EVM and Cosmos.
+ *
+ * @example
+ * ```tsx
+ * import { ARCTIC_1_BANK_PRECOMPILE_ADDRESS } from '@sei-js/evm';
+ * import { ethers } from 'ethers';
+ *
+ * const provider = new ethers.BrowserProvider(window.ethereum); // or any other provider
+ * const signer = await provider.getSigner();
+ *
+ * const accounts = await provider.send('eth_requestAccounts', []);
+ *
+ * const bankPrecompileContract = getBankPrecompileEthersV6Contract(ARCTIC_1_BANK_PRECOMPILE_ADDRESS, signer);
+ *
+ * const balance = await bankPrecompileContract.balance(accounts[0], 'usei');
+ * ```
+ *
+ * @param precompileAddress The 0X address of the precompile contract.
+ * @param signer The 'ethers' signer to be used with the contract.
+ * @returns The typed contract instance allowing interaction with the precompile contract.
+ * @category Cosmos Interoperability
+ */
+export const getBankPrecompileEthersV6Contract = (precompileAddress: `0x${string}`, signer: ethers.Signer): BankPrecompileContract => {
```

Comment:
is there a way to query the bank precompile without connecting it with a signer? in the case that a developer just wants to query and not sign anything